### ISSUE TITLE
MongoDB example LB with hostname and same port 27017

### ISSUE THREAD

**Issue by rahulmr** [little-angry-clouds/haproxy-network-ingress#22]
Hello, Thanks for the project.
I was wondering how to use this for mongodb.
If you get time please answer my query.

I have a scenario where I need to deploy mongodb in multiple namespaces and pods can communicate with each other like mongodb is being accessed by other pods using svc name mongo.namepsace.svc.cluster.local
But now my manager want me to expose mongodb to outside world so that anyone can connect using Studio 3T.

So basically I need a dnsname and port shuld be same 27017. since I want to create this dynamically no manual tasks.

Basically it TCP stream
